### ISSUE TITLE
Quantization code in different path to where specified in Help Documentation

### ISSUE THREAD

**Issue by alex-birch** [tensorflow/tensorflow#6181]
Following the documentation here to Quantize - https://www.tensorflow.org/versions/r0.12/how_tos/quantization/index.html

seems to be wrong.

I had to change  
`bazel build tensorflow/contrib/quantization/tools:quantize_graph
bazel-bin/tensorflow/contrib/quantization/tools/quantize_graph \
--input=/tmp/classify_image_graph_def.pb \
--output_node_names="softmax" --output=/tmp/quantized_graph.pb \
--mode=eightbit`
TO
`bazel build tensorflow/tools/quantization/quantize_graph
bazel-bin/tensorflow/tools/quantization/quantize_graph \
--input=/tmp/classify_image_graph_def.pb \
--output_node_names="softmax" --output=/tmp/quantized_graph.pb \
--mode=eightbit
`
ie it seems like it's moved from tensorflow/contrib to tensorflow/tools/

**Comment by michaelisard** [comment]
Thanks! Would you submit a PR with the documentation fix and reference this issue?

**Comment by jart** [comment]
Just sent out a PR so this will be fixed shortly.
